### PR TITLE
Updated error messaging when ctx use/update is used for vanilla clusters

### LINKED ISSUE #2957: ability to add or delete contexts for vanilla kubernetes clusters using `okteto ctx` subcommands
**Is your feature request related to a problem? Please describe.**
Currently `okteto ctx use` and `okteto ctx delete` don't work for vanilla k8s clusters.

**Describe the solution you'd like**
To be able to add or delete vanilla clusters using the same `okteto ctx` commands which work for clusters with Okteto installed. Or at least we should have the CLI tell us that vanilla contexts are managed through the `kubeconfig`.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by vimalpatel19**
Signed-off-by: Vimal Patel <vimalpatel0611@gmail.com>

# Proposed changes

Fixes #2957

Added more descriptive error messaging for when a user attempts to run 'okteto ctx' to use/delete for a vanilla cluster.

### CODE REVIEW

File: cmd/context/create.go [hunk 1]
```diff
@@ -128,7 +128,8 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 		} else {
 			if !isValidCluster(ctxOptions.Context) {
 				return oktetoErrors.UserError{E: fmt.Errorf("invalid okteto context '%s'", ctxOptions.Context),
-					Hint: "Please run 'okteto context' to select one context"}
+					Hint: fmt.Sprintf("Please run 'okteto context' to select one context. Please run 'kubectl config use-context %s' if attempting to use a Kubernetes context that is a not an Okteto context", ctxOptions.Context),
+				}
```
Comment (pchico83) [inline cmd/context/create.go:132]: I would leave the previous message. You can select kubernetes context with `okteto ctx use`

File: cmd/context/delete.go [hunk 2]
```diff
@@ -57,7 +58,10 @@ func Delete(okCtx string) error {
 				validOptions = append(validOptions, k)
 			}
 		}
-		return fmt.Errorf("'%s' context doesn't exist. Valid options are: [%s]", okCtx, strings.Join(validOptions, ", "))
+		return oktetoErrors.UserError{
+			E:    fmt.Errorf("'%s' context doesn't exist. Valid options are: [%s]", okCtx, strings.Join(validOptions, ", ")),
+			Hint: fmt.Sprintf("Please run 'kubectl config delete-context %s' if attempting to delete a Kubernetes context that is a not an Okteto context", okCtx),
```
Comment (pchico83) [inline cmd/context/delete.go:63]:
```suggestion
			Hint: fmt.Sprintf("To delete a Kubernetes context run 'kubectl config delete-context %s'", okCtx),
```

**Review by pchico83 — APPROVED** [review]

File: cmd/context/create.go (hunk 1 above)
Comment (vimalpatel19) [reply cmd/context/create.go]: Done

File: cmd/context/delete.go (hunk 2 above)
Comment (vimalpatel19) [reply cmd/context/delete.go]: @pchico83 updated as per your suggestion.

**Review by pchico83 — APPROVED** [review]

**Comment by pchico83** [comment]
Awesome @vimalpatel19 , thanks for your contribution!

**Pull request merged**